Modelica/Electrical/Polyphase/Basic/MultiDelta.mo

```diff
@@ -1,24 +1,31 @@
 within Modelica.Electrical.Polyphase.Basic;
 model MultiDelta
   "Delta (polygon) connection of polyphase systems consisting of multiple base systems"
+  import Modelica.Electrical.Polyphase.Functions.numberOfSymmetricBaseSystems;
+  import Modelica.Utilities.Streams.print;
   parameter Integer m(final min=2) = 3 "Number of phases";
-  final parameter Integer mSystems=
-      Polyphase.Functions.numberOfSymmetricBaseSystems(
-      m) "Number of base systems";
-  final parameter Integer mBasic=integer(m/mSystems)
-    "Phase number of base systems";
-
+  parameter Integer kPolygon=1 "Alternative of polygon";
+  final parameter Integer mSystems=numberOfSymmetricBaseSystems(m) "Number of base systems";
+  final parameter Integer mBasic=integer(m/mSystems) "Phase number of base systems";
   Polyphase.Interfaces.PositivePlug plug_p(final m=m)
     annotation (Placement(transformation(extent={{-110,-10},{-90,10}})));
   Polyphase.Interfaces.NegativePlug plug_n(final m=m)
     annotation (Placement(transformation(extent={{90,-10},{110,10}})));
+protected
+  parameter Integer kP=if (mBasic<=2 or kPolygon<1 or kPolygon>integer(mBasic - 1)/2) then 1 else kPolygon;
 equation
+  when initial() then
+    if (mBasic<=2 or kPolygon<1 or kPolygon>integer(mBasic - 1)/2) then
+      print("MultiDelta: replaced erroneous kPolygon = "+String(kPolygon)+" by kPolygon = 1");
+    end if;
+  end when;
```

Comment:
Not sure, if I like this kind of verbosity. In my opinion it would be good enough to have it documented only. Also, setting min=1 for kPolygon might be useful.

Reply:
removed "print", inserted a line in dicumentation